.travis.yml

```diff
@@ -1,5 +1,9 @@
 sudo: false
 language: scala
+stages:
+  - name: test
+  - name: release
+    if: (branch = master AND type = push) OR (tag IS present)
 matrix:
```

Comment:
I think this needs to be `stages` to use stages

Reply:
You mean `jobs`? I was under the impression that `jobs` and `matrix` were equivalent (and [this](https://travis-ci.org/lihaoyi/fastparse/builds/524877277) goes in that sense).

Reply:
Just renamed it to `jobs` just in case.

Reply:
I meant `jobs` yes, but maybe it's equivalent to `matrix`, I don't know 🤔